activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -112,14 +113,15 @@ def change_table(table_name, options = {}) # :nodoc:
 
         module StraightReversions # :nodoc:
           private
-            { transaction:       :transaction,
-              execute_block:     :execute_block,
-              create_table:      :drop_table,
-              create_join_table: :drop_join_table,
-              add_column:        :remove_column,
-              add_timestamps:    :remove_timestamps,
-              add_reference:     :remove_reference,
-              enable_extension:  :disable_extension
+            { transaction:           :transaction,
```

Comment:
Why was every line in this hash changed when you only added a single line?

Reply:
@sgrif  because it's very clearly [already] aligned to the longest key?

(though it does seem to have ended up one space further over than it should be.)

Reply:
Sorry, I should have been more clear on my intent:

Do you think we could drop the visual indent change here so we aren't changing every line in this hash? I think having the one line that is not indented the same is fine.